src/adapters/sonobi.js

```diff
@@ -0,0 +1,87 @@
+var bidfactory = require('../bidfactory.js');
+var bidmanager = require('../bidmanager.js');
+var utils = require('../utils');
+
+var SonobiAdapter = function SonobiAdapter(){
+  var trinity_ep = 'http://apex.go.sonobi.com/trinity.js?key_maker=';
+  var test = false; //  tag tester = true || false
+  var cb_map = {};
+
+  function _phone_in(params){
+    var bids = params.bids || [];
+    _insert_header(_keymaker(bids));
+  }
+
+  function _insert_header(keyring){ //  Inject trinity into header
```

Comment:
Usage is `adloader.loadScript('scriptUrl', cb, cacheUrl)` 
For JSONP you can just append your callback to `scriptUrl`